scripts/simplify_network.py

```diff
@@ -110,21 +110,21 @@
 logger = logging.getLogger(__name__)
 
 
-def simplify_network_to_380(n, linetype):
+def simplify_network_to_base_voltage(n, linetype, base_voltage):
     """
-    Fix all lines to a voltage level of 380 kV and remove all transformers.
+    Fix all lines to a voltage level of base voltage level and remove all transformers.
     The function preserves the transmission capacity for each line while updating
     its voltage level, line type and number of parallel bundles (num_parallel).
     Transformers are removed and connected components are moved from their
     starting bus to their ending bus. The corresponding starting buses are
     removed as well.
     """
-    logger.info("Mapping all network lines onto a single 380kV layer")
+    logger.info(f"Mapping all network lines onto a single {int(base_voltage)}kV layer")
 
-    linetype_380 = linetype
-    n.lines["type"] = linetype_380
-    n.lines["v_nom"] = 380
-    n.lines["i_nom"] = n.line_types.i_nom[linetype_380]
+    linetype_base = linetype
```

Comment:
let's remove line 124. It's not needed

Reply:
Ok, I will do that, as you have mentioned very redundant piece.